### PR TITLE
test: add visual tests to donations

### PR DESCRIPTION
**Pull request by NoamGaash**
# Description
1. adds a test for the donation modal
2. adds a n aria attribute to the modal so it will be considered as a dialog

### CODE REVIEW

**Comment by NoamGaash** [comment]
Hi @YaelChen , would you like to review the code? tell me if there's something I can make more clear or readable, or if anything requires adding more comment or something like this

File: src/pages/DonateModal/DonateModal.tsx [hunk 1]
```diff
@@ -24,6 +24,9 @@ const DonateModal: React.FC<DonateModalProps> = ({ isVisible, onClose }) => {
     <Modal
       open={isVisible}
       onClose={onClose}
+      style={{ color: '#1498e5' }}
```
Comment (YaelChen) [inline src/pages/DonateModal/DonateModal.tsx:27]: Shouldn't this be removed? Isn't that what I temporarily added before the dark mode text color fix?

**Comment by YaelChen** [comment]
@NoamGaash seems clear and readable, but I can't run the tests as they're skipped. I need to try to update the thing you once mentiond to be able to run the visual tests. I'll look for that comment to try to figure that out.

File: src/pages/DonateModal/DonateModal.tsx (hunk 1 above)
Comment (NoamGaash) [reply src/pages/DonateModal/DonateModal.tsx:27]: Thank you! You're right, I missed that while solving the merge conflict. Fixed 🙏

**Comment by NoamGaash** [comment]
@YaelChen it's not mandatory to run the tests, the interesting part is here - 
image

**Review by YaelChen — APPROVED** [review]
Seems great then, I'll aproove and try run tests myself later, as I got installation errors 😅

**Pull request merged**